.github/workflows/tagged_release.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: "tagged-release"
+
+on:
+  push:
+    tags:
+      - "v*"
```

Comment:
I assume this is the wildcard that triggers the job to create a release note?

Reply:
Yes, this action will run on any tag that starts with "v".